components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/load-service-provider.jsp

```diff
@@ -31,6 +32,13 @@
 <%@ page import="org.owasp.encoder.Encode" %>
 <%@ page import="org.wso2.carbon.identity.application.mgt.ui.ApplicationBean"%>
 <%@ page import="org.wso2.carbon.identity.application.mgt.ui.util.ApplicationMgtUIUtil"%>
+<%@ page import="org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementService" %>
+<%@ page import="org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementServiceImpl" %>
+<%@ page import="org.wso2.carbon.context.PrivilegedCarbonContext" %>
+<%@ page import="org.wso2.carbon.identity.claim.metadata.mgt.model.ClaimDialect" %>
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.ArrayList" %>
+<%@ page import="java.util.stream.Collectors" %>
```

Comment:
Why backend services had been imported here.
Please approach the client server model.
This will add a dependency of backend modules for the UI module